.github/workflows/update-mise-tools.yaml

```
@@ -9,7 +9,7 @@ name: Update mise tool versions
 permissions: {}
 
 jobs:
-  updatecli_major:
+  update-mise-tools:
```

```suggestion
  update_mise_tools:
```

Maybe we want underscores here? Forgot which one is the ideal/standard

---

yeah i think underscores!